packages/shared/src/lib/flows/workflow-import-schema.ts

```diff
@@ -0,0 +1,58 @@
+import { Static, Type } from '@sinclair/typebox';
+import { TypeCompiler } from '@sinclair/typebox/compiler';
+import { FlowTemplateMetadata } from './dto/flow-template-request';
+import { Trigger } from './triggers/trigger';
+
+export const WorkflowTemplateSchema = Type.Object({
+  displayName: Type.String(),
+  trigger: Trigger,
+  valid: Type.Boolean(),
+});
+
+export type WorkflowTemplate = Static<typeof WorkflowTemplateSchema>;
+
+export const WorkflowImportSchema = Type.Composite([
+  Type.Omit(FlowTemplateMetadata, [
+    'id',
+    'type',
+    'projectId',
+    'organizationId',
+  ]),
+  Type.Object({
+    template: WorkflowTemplateSchema,
+  }),
+]);
+
+export type WorkflowImport = Static<typeof WorkflowImportSchema>;
+
+export const workflowImportValidator =
+  TypeCompiler.Compile(WorkflowImportSchema);
+
+export const triggerValidator = TypeCompiler.Compile(Trigger);
```

Comment:
I want the trigger validation to be more complete. 
For example:
nextAction is currently any https://github.com/openops-cloud/openops/blob/main/packages/shared/src/lib/flows/triggers/trigger.ts#L17C3-L17C41
it can be only values here
packages/shared/src/lib/flows/actions/action.ts